resources/views/pages/study/representation/faculty-council.blade.php

```diff
@@ -45,6 +35,22 @@
 		<br>
 		
 	</div>
+	<div class="col">
+		<iframe width="560" height="315" src="https://www.youtube.com/embed/h0yKz8iRgbY" frameborder="0" allowfullscreen></iframe>
+		<br>
```

Comment:
Instead of adding all of these `<br>` tags you could add some margin or padding to the `<iframe>`. Bootstrap has some helpers for this, see [spacing](https://v4-alpha.getbootstrap.com/utilities/spacing/).